.github/workflows/release-fleet.yml

```diff
@@ -28,15 +28,14 @@ jobs:
             }
 
       - name: Checkout detection-rules
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           path: detection-rules
 
       - name: Checkout elastic/integrations
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          token: ${{ secrets.PROTECTIONS_MACHINE_TOKEN }}
-          ref: ${{github.event.inputs.target_branch}}
```

Comment:
Removing the reference to the master seemed to help fetch the integrations repo successfully.

Reply:
so strange - I wish we understood why